spot_driver/spot_driver/ros_helpers.py

```diff
@@ -328,3 +330,46 @@ def lookup_a_tform_b(
                 raise e
             time.sleep(0.01)
     return None
+
+
+class TriggerServiceWrapper:  # TODO: caching of spot_ros might be funky with multiple instances up
+    """A descriptor for calling callbacks for trigger services"""
+
+    service_func: Callable[[SpotWrapper], Trigger.Response]
+    service_name: str
+    args: Optional[tuple] = ()
+    kwargs: Optional[dict | None] = None
+
+    def __init__(
+        self, service_func: Callable, service_name: str, *args: Optional[Any], **kwargs: Optional[Any | None]
```

Comment:
```suggestion
        self, service_func: Callable, service_name: str, *args: Optional[Any], **kwargs: Optional[Any]
```

nit: does `Any` cover the `None` type?